src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -127,17 +130,21 @@ public void setPerson(Person target, Person editedPerson) {
      */
     @Override
     public ObservableList<Person> getFilteredPersonList() {
```

Comment:
the method name and javadocs have to change to reflect that this is a sorted + filtered list now

Reply:
I changed the javadoc in the `Model` interface